easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.3-foss-2020a-Python-3.8.2.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'netcdf4-python'
+version = '1.5.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://unidata.github.io/netcdf4-python/'
+toolchain = {'name': 'foss', 'version': '2020a'}
+description = """Python/numpy interface to netCDF."""
+
+toolchainopts = {'usempi': True}
```

style nit: move toolchain down:
```suggestion
description = """Python/numpy interface to netCDF."""

toolchain = {'name': 'foss', 'version': '2020a'}
toolchainopts = {'usempi': True}
```

---

fixed. Thanks